### DISCUSSION TITLE
Can export a single array like json?

### DISCUSSION THREAD

**Discussion by cangSDARM** [toml-lang/toml#972] [Q&A] [answered]
in json we can write like this:
```json
[
"arrayItem1",
"arrayItem2"
]
```
It is a valid json file that contains only one array.

Is there a way to do this in toml like json, exporting a single array?

**Comment by jakedane** [comment, marked as answer]
No, you must write it as a key/value pair. The key can not be omitted.

The specification says it's valid to have an empty (quoted) key, but it is discouraged. So this would be valid but discouraged:
```toml
"" = [
  "arrayItem1",
  "arrayItem2"
]
```
And it doesn't get you what you want because in json this would translate to:
```
{
  "": [
    "arrayItem1",
    "arrayItem2"
  ]
}
```
